src/pages/status-change-event-page/status-change-event-validation.ts

```diff
@@ -1,14 +1,16 @@
 import Joi, { ValidationErrorItem } from "joi";
 import { ValidateFormParams } from "../FormPageHoc";
-import { eventSchema } from "../event-schema";
+import { startDateSchema } from "../stage-schema";
 
-const statusChangeEventSchema = eventSchema.concat(
-  Joi.object({
-    developmentPlanEvent: Joi.required().messages({
-      "any.required": "Select the status of your Local Plan",
-    }),
-  })
-);
+const statusChangeEventSchema = Joi.object({
+  eventDate: startDateSchema,
+  notes: Joi.string().allow("").max(100).messages({
```

Comment:
I think 100 is the old character limit we have changed that to 200 now. This could be shared with the schema we are using for notes in stageSchema so the 2 will always be the same